src/scores/utils.py

```diff
@@ -216,3 +216,10 @@ def check_dims(xr_data: XarrayLike, expected_dims: list[str], mode: Optional[str
                     f"Dimensions {list(xr_data[data_var].dims)} of data variable "
                     f"'{data_var}' are not {mode} to the dimensions {sorted(dims_set)}"
                 )
+
+
+class InvalidInputError(Exception):
```

Comment:
We've just been using `ValueError` in scores. I suggest updating to `ValueError` whenever `InvalidInputError` is used.

Reply:
Thanks. It has now been resolved.